src/main/java/io/cdap/delta/bigquery/AvroEventWriter.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.delta.bigquery;
+
+import io.cdap.cdap.api.data.format.StructuredRecord;
+import org.apache.avro.file.DataFileWriter;
+import org.apache.avro.io.DatumWriter;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+/**
+ * EventWriter that writes records in Avro format.
+ */
+public class AvroEventWriter implements EventWriter {
+  private DataFileWriter<StructuredRecord> avroWriter;
+
+  AvroEventWriter(org.apache.avro.Schema avroSchema, OutputStream outputStream) {
+    DatumWriter<StructuredRecord> datumWriter = new RecordDatumWriter();
+    try {
+      avroWriter = new DataFileWriter<>(datumWriter).create(avroSchema, outputStream);
+    } catch (IOException e) {
+      e.printStackTrace();
```

Comment:
do we need to handle the exception here by logging the issue and throw a runtime exception ?

Reply:
ah.. forgot to change it. yeah we should throw exception from here. fixed the code.